src/screentovnc.cpp

```diff
@@ -179,21 +203,21 @@ ScreenToVnc::ScreenToVnc(QObject *parent, bool smoothScaling, float scalingFacto
 
     m_server->newClientHook = newclient;
 
-    if (m_doMouseHandling){
+    if (m_doMouseHandling) {
         m_server->ptrAddEvent = mouseHandler;
     }
 
-    if (m_doKeyboardHandling){
+    if (m_doKeyboardHandling) {
         uinputCreateKeyboardDevice();
         m_server->kbdAddEvent = keyboardHandler;
     }
 
     // check if launched by systemd with a ready socket (LISTEN_FDS env var)
     int sd_fds = sd_listen_fds(1);
-    if (sd_fds){
-        for (int i = SD_LISTEN_FDS_START; i <= (SD_LISTEN_FDS_START + sd_fds - 1); i++){
+    if (sd_fds) {
+        for (int i = SD_LISTEN_FDS_START; i <= (SD_LISTEN_FDS_START + sd_fds - 1); i++) {
```

Comment:
eh? nit: could be: i < SD_LISTEN_FDS_START + sd_fds

Reply:
True.